app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+class IncorrectCommand(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    if len(command) != 3 or "cp" not in command:
+        raise IncorrectCommand("Incorrectly written command")
```

Comment:
This can be simplified to one if statement. Remove unnecessary return and raises. Instead check if command is correct i one if and then nest the code under this condition if it is satisfied, if it's not satisfied - don't do anything.